newsplease/pipeline/pipelines.py

```diff
@@ -452,10 +452,10 @@ def __init__(self):
             # check if the necessary indices exist and create them if needed
             if not self.es.indices.exists(self.index_current):
                 self.es.indices.create(index=self.index_current, ignore=[400, 404])
-                self.es.indices.put_mapping(index=self.index_current, doc_type='article', body=self.mapping)
+                self.es.indices.put_mapping(index=self.index_current, doc_type='_doc', body=self.mapping)
```

Comment:
why did you change from article to _doc?

Reply:
So, this is not a must, but according to : https://github.com/elastic/elasticsearch/pull/27816
It’s the right way to go.